src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  box-sizing: border-box;
+
+  &__photo {
+    background-image: url(..//images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__tittle {
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 40px;
+    margin-bottom: auto;
+    color: rgba(6, 11, 53, 1);
+    font-weight: 500;
+  }
+
+  &__prodact-code {
```

Comment:
There is a typo in the class name '&__prodact-code'. It should be '&__product-code' to match the corrected HTML class name.